### PR TITLE
Create vaibhav.json

### PR DESCRIPTION
**Pull request by Vaibhav-Kashyap**

### CODE REVIEW

File: domains/vaibhav.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "Vaibhav-Kashyap",
+    "email": "",
+    "discord": "Vaibhav Kashyap#5747"
+  },
+  "record": {
+
```
Comment (TheBotlyNoob) [inline domains/vaibhav.json:8]: Please add at least 1 record.

**Review by TheBotlyNoob — DISMISSED** [review]

File: domains/vaibhav.json (hunk 1 above)
Comment (Vaibhav-Kashyap) [reply domains/vaibhav.json:8]: How do I edit it? or do I have to make a new pull request?

Comment (Vaibhav-Kashyap) [reply domains/vaibhav.json]: Nvm There I fixed it.

File: domains/vaibhav.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "Vaibhav-Kashyap",
+    "email": "",
+    "discord": "Vaibhav Kashyap#5747"
```
Comment (MaskDuck) [inline domains/vaibhav.json:5]: is this account a valid one?

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/vaibhav.json (hunk 2 above)
Comment (MaskDuck) [reply domains/vaibhav.json:5]: @Vaibhav-Kashyap

Comment (Vaibhav-Kashyap) [reply domains/vaibhav.json:5]: Yes, it's a valid user.

Comment (Vaibhav-Kashyap) [reply domains/vaibhav.json:5]: @MaskDuck

Comment (Vaibhav-Kashyap) [reply domains/vaibhav.json:5]: @MaskDuck ?

Comment (MaskDuck) [reply domains/vaibhav.json:5]: no it isnt

Comment (MaskDuck) [reply domains/vaibhav.json:5]: i checked through bot

Comment (MaskDuck) [reply domains/vaibhav.json:5]: ah it is

**Review by MaskDuck — APPROVED** [review]
not sus. btw wanna join [our discord server](https://discord.gg/dJnGjeWMHx)?

File: domains/vaibhav.json (hunk 2 above)
Comment (Vaibhav-Kashyap) [reply domains/vaibhav.json:5]: Ye

Comment (Vaibhav-Kashyap) [reply domains/vaibhav.json:5]: Can it be approved now?

**Review by mtgsquad — APPROVED** [review]

**Pull request merged**